Add note on context support in the step-ca command.

tashian opened the pull request:
This is a rare moment where it seems to make sense to mention the step-ca command in the step-cli docs, in order to house all of the docs around contexts in the same place.

maraino commented on step-ca/configuration.mdx:
````diff
@@ -616,7 +616,7 @@ $ step ca provisioner add Google --type oidc --ca-config $(step path)/config/ca.
 ```
 
-Example `ca.json` provisioner configuration for a G-suite provisioner:
+Example `ca.json` provisioner configuration for a G Suite provisioner:
````
Ok, but they renamed to Google Workspace.

tashian committed:
Add note on context support in the step-ca command.

This is a rare moment where it seems to make sense to mention the step-ca command in the step-cli docs, in order to house all of the docs around contexts in the same place.

tashian committed:
Merge branch 'main' into step-ca-contexts

tashian commented:
@dopey just wanted to bump this
